src/routes/liking.route.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import LikingsController from '../controllers/likings.controller';
+import LikingMiddleware from '../middlewares/liking.middleware';
+import UserMiddleware from '../middlewares/userMiddlware';
+
+const app = express.Router();
+
+const { checkToken } = UserMiddleware;
+const { checkUserLiking } = LikingMiddleware;
+const { addLikeAccommdation } = LikingsController;
+
+app.post('/:accommodationId', checkToken, checkUserLiking, addLikeAccommdation);
+
+export default app;
```

Comment:
Why don't you put this into the accommodation route. Don't think it's important to create another file. I suggest to have a route like this : /accommodations/accommodationId/like or /accommodations/accommodationId/unlike

Reply:
Thank you @lemoissonM  for your feedback. I already worked on that